book/website/collaboration/inclusive-events/inclusive-events-schedules.md

```diff
@@ -0,0 +1,206 @@
+
+It is crucial in your planning to prioritize inclusivity and accessibility in your event's scheduling and programming, and to recognize that the many of the traditional approaches to schedules and agendas may create barriers or be inaccessible to people with lived experience or disabilities. 
+
+
+## Event Design: Timing and Event Duration
+When developing your event's program or agenda, you should consider the timing of your event's content. This includes identifying when your event will start and end, the duration of your event's content and how you will structure the event content to make it accessible, and finally the duration of the event itself.
+
+### Start and End Times
+You may want to consider planning sessions and event start times around travel, for example by starting later in the day and finishing mid afternoon. This can help support attendes who need to travel to attend your event and may not be comfortable traveling at peak times. 
+If your event needs to start early in the morning, consider hosting a welcome session that is able to be missed if necessary. 
+
+Additionally, consider planning your event around important holidays and cultural celebrations. By considering when your attendees will be required elsewhere, you will be more likely to find a time that suits everyone. 
+
+### Session Duration
+Your schedule should take into account breaks, meal times, and carefully plan the length of its sessions. 
+
+If your event takes place in person, make sure to allow for sufficient time for people with lived experience and mobility difficulties to move between locations. For example, consider how long it would take to get to an accessible elevator and then travel between floors, or how far away accessible restrooms are. If your main room has only one exit and a large audience, it may take significantly longer to exit the room and cause attendees stress as they have to rush between sessions. 
+
+Additionally, many people who experience cognitive or physical fatigue find it difficult to actively engage in events that last longer than 40 minutes without a meaningful break. 
+
+This timing can vary depending on individuals, however both ADHD advocates and people with a lived experience of cognitive impairements (such as stroke, traumatic brain injury, and brain fog) have advocated for 30-40 minute session blocks for activities that require sustained attention and engagement. 
+
+Try to plan for the most important parts of your event's content being shared within a blocks of time that are under 40 minutes.
+
+Because many events plan in 1 hour or 1.5 hour timing blocks, consider choosing shorter sessions or incorporating breaks within your session. This can be  as well as and allowing for 5-10 minutes for an introduction and session set up. This can help reduce the cognitive strain that many people with lived experience of cognitive impairements are affected by. 
```

Comment:
This can be  as well as ~~and~~ allowing for 5-10 minutes for an introduction and session set up. This can help reduce the cognitive strain that many people with lived experience of cognitive **impairments** are affected by.